src/nft/utils/transactionResponse.ts

```diff
@@ -54,7 +54,7 @@ export const parseTransactionResponse = (transactionResponse: TxResponse | undef
 export const getSuccessfulImageSize = (numSuccessful: number, isMobile: boolean) => {
   const sizeModifier = isMobile ? 2 : 1
   if (numSuccessful === 1) {
-    return 574 / sizeModifier
+    return 474 / sizeModifier
```

Comment:
I think we should make each of these breakpoints into a enum for this function

Reply:
This will all be re-written and standardized when we make the tx complete modal its own page